packages/flutter_test/test/frame_timing_summarizer_test.dart

```diff
@@ -16,10 +16,14 @@ void main() {
       for (int i = 1; i <= 100; i += 1) 1000 * i + 1000,
     ];
     rasterTimes = rasterTimes.reversed.toList();
+    List<int> vsyncStarts = <int>[
+      for (int i = 1; i <= 100; i += 1) -100 * i,
```

Comment:
Nit: having negative timestamp looks strange. Can we change `buildStart` from `0` to some positive numbers so `vsyncStarts` can still be nonnegative?

Reply:
Done.